### PR TITLE
WS-17: Create services

### PR DESCRIPTION
**Pull request by NickOsman11**

### CODE REVIEW

File: backend/Services/LocationService.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+using System;
+using Microsoft.AspNetCore.Cryptography.KeyDerivation;
+using WhaleSpotting.Repositories;
+
+namespace WhaleSpotting.Services
+{
+    public interface ILocationService
+    {
+    }
+    public class LocationService : ILocationService
```
Comment (TimLeach635) [inline backend/Services/LocationService.cs:10]: Line between
```suggestion
    }
    
    public class LocationService : ILocationService
```

File: backend/Services/LocationService.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    {
+        private readonly ILocationRepo _locations;
+
+        public LocationService(ILocationRepo locations)
+        {
+            _locations = locations;
+        }
+    }
+};
```
Comment (TimLeach635) [inline backend/Services/LocationService.cs:19]: I don't think semicolons here are normal!
```suggestion
}
```

File: backend/Services/SightingService.cs [hunk 3]
```diff
@@ -0,0 +1,19 @@
+using System;
+using Microsoft.AspNetCore.Cryptography.KeyDerivation;
+using WhaleSpotting.Repositories;
+
+namespace WhaleSpotting.Services
+{
+    public interface ISightingService
+    {
+    }
+    public class SightingService : ISightingService
```
Comment (TimLeach635) [inline backend/Services/SightingService.cs:10]: Space
```suggestion
    }
    
    public class SightingService : ISightingService
```

File: backend/Services/SightingService.cs [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    {
+        private readonly ISightingRepo _sightings;
+
+        public SightingService(ISightingRepo sightings)
+        {
+            _sightings = sightings;
+        }
+    }
+};
```
Comment (TimLeach635) [inline backend/Services/SightingService.cs:19]:
```suggestion
}
```

File: backend/Services/WhaleService.cs [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+    public class WhaleService : IWhaleService
+    {
+        private readonly IWhaleRepo _whales;
+
+        public WhaleService(IWhaleRepo whales)
+        {
+            _whales = whales;
+        }
+    }
+};
```
Comment (TimLeach635) [inline backend/Services/WhaleService.cs:19]:
```suggestion
}
```

File: backend/Services/WhaleService.cs [hunk 6]
```diff
@@ -0,0 +1,19 @@
+using System;
+using Microsoft.AspNetCore.Cryptography.KeyDerivation;
+using WhaleSpotting.Repositories;
+
+namespace WhaleSpotting.Services
+{
+    public interface IWhaleService
+    {
+    }
+    public class WhaleService : IWhaleService
```
Comment (TimLeach635) [inline backend/Services/WhaleService.cs:10]:
```suggestion
    }
    
    public class WhaleService : IWhaleService
```

File: backend/Startup.cs [hunk 7]
```diff
@@ -63,6 +63,9 @@ public void ConfigureServices(IServiceCollection services)
 
       services.AddTransient<IAuthService, AuthService>();
       services.AddTransient<IUserService, UserService>();
+      services.AddTransient<IWhaleService, WhaleService>();
+      services.AddTransient<ILocationService, LocationService>();
+      services.AddTransient<ISightingService, SightingService>();
```
Comment (TimLeach635) [inline backend/Startup.cs:68]: Very particular of me, but I like these to be in alphabetical order 😅 
```suggestion
      services.AddTransient<IAuthService, AuthService>();
      services.AddTransient<ILocationService, LocationService>();
      services.AddTransient<ISightingService, SightingService>();
      services.AddTransient<IUserService, UserService>();
      services.AddTransient<IWhaleService, WhaleService>();
```

**Review by TimLeach635 — CHANGES_REQUESTED** [review]
Some changes!

File: backend/Services/LocationService.cs (hunk 2 above)
Comment (NickOsman11) [reply backend/Services/LocationService.cs:19]: It's just in the authService file we copied _shrug_

**Review by TimLeach635 — APPROVED** [review]

**Pull request merged**